config/initializers/devise.rb

```diff
@@ -234,10 +234,19 @@
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
 
-  config.omniauth :twitter,
-    Rails.application.secrets.twitter_api_key,
-    Rails.application.secrets.twitter_api_secret
-
+  supported_omniauth_providers = {
+    twitter: [ :twitter_api_key, :twitter_api_secret ],
+    github: [
+      :github_app_id,
+      :github_app_secret,
+      { scope: 'user,user:email' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.